.github/actions/verify-token/package.json

```diff
@@ -26,7 +26,7 @@
     "@actions/github": "5.1.1",
     "@octokit/webhooks-types": "7.3.1",
     "@sigstore/rekor-types": "1.0.0",
-    "sigstore": "1.8.0",
+    "sigstore": "2.2.2",
```

Comment:
Did you mean for this to be `2.2.2`?

Reply:
The package seems to exist at this verison, iiuc https://github.com/sigstore/sigstore-js/releases/tag/sigstore%402.2.2

Reply:
I ask because your PR description says you mean to upgrade to "1.9", so I thought you might have meant `1.9.0`. 
https://github.com/sigstore/sigstore-js/releases/tag/sigstore%401.9.0

Reply:
Yes! I should have updated the title and description. Will do that now. Perhaps, I should have done 2.2.2 in a separate pr. My thinking was that since 2.2.2 was the latest and required only a small change that it would be best to do that here.